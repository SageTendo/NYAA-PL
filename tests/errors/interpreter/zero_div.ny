uWu_nyaa() => yomu(3 supuritto 0);