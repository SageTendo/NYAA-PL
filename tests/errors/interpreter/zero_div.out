Division by zero is not kawaii, please don't do that.